omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml

```diff
@@ -1100,6 +1100,12 @@
                                  answerLabel="LFT"/>
                             <obs conceptId="1271" answerConceptId="1163309"
                                  answerLabel="RFT"/>
+                            <obs conceptId="1271" answerConceptId="1031"
```

Comment:
@reagan-meant tests are added but the display is not very good need to add styling so that they are all displayed properly maybe only 3 per line 

<img width="592" alt="Screenshot 2020-06-29 05 03 04" src="https://user-images.githubusercontent.com/689900/85965618-e632b480-b9c5-11ea-8fdb-d0f473f33698.png">

Reply:
Done

Reply:
https://app.asana.com/0/1164864596604507/1164867478655803